### DISCUSSION TITLE
Blank Page in Production Stage

### DISCUSSION THREAD

**Discussion by adamyulian** [laravel/nova-issues#5813] [General]
I've tinkered with it several times, but I haven't found a solution yet. I have no problem in local stage, but when i move to production stage i have problem start from very beginning installation, https://mydomain.com/public/nova/login page that only appears is a blank page, not a 404 but an empty/blank page. I tried to read the documentation regarding the gate but after I tried it still didn't work. Can you guys help me?

**Comment by crynobone** [comment]
Laravel doesn't support such usage: https://laravel.com/docs/10.x/installation#directory-configuration
